_data/trans/dk.json

```diff
@@ -0,0 +1,47 @@
+{
+    "about": "Om",
+    "contribute": "Bidrag på GitHub",
+    "defaultnote_easy": "Besøg den linkede adresse og slet din konto.",
+    "defaultnote_email": "Send en email til den angivne adresse og anmod om at få slettet din konto.",
+    "difficulty": "sværhedsgrad",
+    "difficulty_easy": "let",
+    "difficulty_hard": "svær",
+    "difficulty_impossible": "umulig",
+    "difficulty_limited": "begrænset mulighed",
+    "difficulty_medium": "medium",
+    "extension_browser": "Chrome/Firefox Udvidelse",
+    "extensionguide": "Udvidelse Prik Guide",
+    "extensionp1": "Når du er på en hjemmeside, der er registreret i JustDeleteMe, så vil udvidelsen tilføje en lille prik i din browsers udvidelsesfelt. Hvis du klikker på denne prik vil du blive sendt videre til adressen hvor du kan slette din konto.",
+    "extensionp2": "For at installere udvidelsen, skal du blot fortsætte til",
+    "extensionp3": "Chrome Web Store (Uofficiel)",
+    "extensionp4": "Firefox Add-ons",
+    "extensionp5": "",
+    "footercredits": "Lavet af",
+    "guide": "Guide",
+    "guideeasy": "Kontoen kan nemt slettes",
+    "guideexplanations": "Linksene er farvekodede for at angive, hvor svært det er at slette sin konto fra webtjenesterne:",
+    "guidehard": "Kontoen kan ikke slettes uden at kontakte kundeservice",
+    "guideimpossible": "Kontoen kan ikke slettes",
+    "guidelimited": "Kontoen kan kun slettes, hvis du bor i et land med specifikke privatlivsrettigheder",
+    "guidemedium": "Kontoen kan slettes, men det kræver flere trin.",
+    "hideinfo": "skjul info...",
+    "jgmd": "Besøg Just Get My Data",
+    "name": "Dansk",
+    "noinfo": "Ingen oplysninger tilgængelige",
+    "noresults": "Kan du ikke finde det, du leder efter?",
+    "noresultshelp": "Hjælp med at gøre JustDeleteMe bedre",
+    "popular": "Populær",
+    "pullrequest": "Opret en Pull Request på GitHub",
+    "reset": "Nulstil",
+    "search": "Søg",
+    "sendmail": "Send email",
```

Comment:
Same question

Reply:
Yes, "send email" means the same in Danish.